lib/scripts/services/contextMenu.js

```diff
@@ -1,19 +1,57 @@
+function _checkIfWordCountNotOne(inputString){
+    var s = inputString;
+    s = s.replace(/(^\s*)|(\s*$)/gi,"");
+    s = s.replace(/[ ]{2,}/gi," ");
+    s = s.replace(/\n /,"\n");
+    if (s.split(' ').length != 1){
+        alert('Please select a single word only. Phrases such as: "' + inputString + '" aren\'t allowed');
+        return true;
+    }
+    return false;
+}
+
+function _currentURLIsBlacklistedOrMTWIsOff(documentURL) {
+    var promise = new Promise(function(resolve, reject) {
+      chrome.storage.local.get(['activation', 'blacklist'], function(result) {
+        var blacklistWebsiteReg = new RegExp(result.blacklist);
+        var activation = result.activation;
+        if (activation == false) {
+          alert('MindTheWord is switched off');
```

Comment:
Here we should also instruct the user how to switch MTW on. Otherwise some users might think that MTW is switched off because of a bug.